### PR TITLE
Delay L1 transacting when the gas price is high

### PR DESCRIPTION
**Pull request by PlasmaPower**

### CODE REVIEW

File: packages/arb-node-core/staker/staker.go [hunk 1]
```diff
@@ -114,7 +122,59 @@ func (s *Staker) RunInBackground(ctx context.Context) chan bool {
...
+		}
+	}
+	latestBlockInfo, err := s.client.BlockInfoByNumber(ctx, nil)
+	if err != nil {
+		logger.Warn().Err(err).Msg("error getting latest block")
+		return true
+	}
+	latestBlockNum := latestBlockInfo.Number.ToInt()
+	if s.lastActCalledBlock == nil {
+		s.lastActCalledBlock = latestBlockNum
```
Comment (hkalodner) [inline packages/arb-node-core/staker/staker.go:144]: Seems like `lastActCalledBlock` should also be updated when the validator takes an action, but I don't think it does currently

**Review by hkalodner — CHANGES_REQUESTED** [review]

File: packages/arb-node-core/staker/staker.go (hunk 1 above)
Comment (PlasmaPower) [reply packages/arb-node-core/staker/staker.go:144]: I completely missed this, thanks

Comment (PlasmaPower) [reply packages/arb-node-core/staker/staker.go:144]: Specifically, it should be updated not only when the validator takes an action, but also when it doesn't, as the highGasBlocksBuffer is still updated. I've fixed this.

**Review by hkalodner — APPROVED** [review]
LGTM

**Pull request merged**